reactor-core/src/main/java/reactor/core/publisher/FluxMetrics.java

```diff
@@ -295,9 +295,8 @@ static Tags resolveTags(Publisher<?> source, Tags tags) {
 		Scannable scannable = Scannable.from(source);
 
 		if (scannable.isScanAvailable()) {
-			List<Tag> discoveredTags = scannable.tagsDeduplicated()
```

Comment:
The method `Scannable#tagsDeduplicated()` is now unused, but I didn't drop it, as it's part of a public interface.

Reply:
Deprecating the method SGTM; indeed a [GitHub search](https://github.com/search?q=tagsDeduplicated&type=code) does not seem to turn up additional usages. I've proposed a `@deprecated` sentence that hopefully explains the situation, without claiming that no caller would find the method useful.